### PR TITLE
Https

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

File: httpd.conf [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+#
+# Listen: Allows you to bind Apache to specific IP addresses and/or
+# ports, instead of the default. See also the <VirtualHost>
+# directive.
+#
+# Change this to Listen on specific IP addresses as shown below to 
+# prevent Apache from glomming onto all bound IP addresses.
+#
+#Listen 12.34.56.78:80
+Listen 80
```
Comment (miczyg1) [inline httpd.conf:52]: @macpijan doesn't it need to listen on 443?

Comment (macpijan) [reply httpd.conf:52]: @miczyg1 It is the default config with changes described in the https://hub.docker.com/_/httpd to enable HTTPS. I haven't really analyzed that closer.

Comment (macpijan) [reply httpd.conf:52]: I believe that including `conf/extra/httpd-ssl.conf` may override this setting.

Comment (miczyg1) [reply httpd.conf:52]: @macpijan probably. Don't we need to add it explicitly?

Comment (macpijan) [reply httpd.conf:52]: @miczyg1 It already works with the `Include conf/extra/httpd-ssl.conf` directive and the Listen 443 is defined here: https://github.com/finalcut/config/blob/master/windows/Apache/conf/extra/httpd-ssl.conf#L37 so it's not really a requisite to have this up and running properly.

We can, however, adjust and simplify the config to our needs if we feel like doing so.

Comment (macpijan) [reply httpd.conf:52]: Probably advised in case of long-term maintenance.

**Comment by pietrushnic** [comment]
@miczyg1 @macpijan is there any problem with merging this changes?

**Review by miczyg1 — APPROVED** [review]

**Comment by miczyg1** [comment]
@pietrushnic actually no. It is a dev server, so if it will be any improvement, fix to be applied. It can be done in another PR. This looks good to me.

**Pull request merged**